precompiles/outposts/osmosis/types.go

```diff
@@ -0,0 +1,219 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package osmosis
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"golang.org/x/exp/slices"
+
+	"github.com/cosmos/btcutil/bech32"
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	"github.com/ethereum/go-ethereum/common"
+	cmn "github.com/evmos/evmos/v15/precompiles/common"
+)
+
+const (
+	/// MaxSlippagePercentage is the maximum slippage percentage that can be used in the
+	/// definition of the slippage for the swap.
+	MaxSlippagePercentage uint8 = 20
+	/// MaxWindowSeconds is the maximum number of seconds that can be used in the
+	/// definition of the slippage for the swap.
+	MaxWindowSeconds uint64 = 60
```

Comment:
I think was suppose to be 10 right?
```suggestion
	MaxWindowSeconds uint64 = 10
```

Reply:
Isn't too low a twap period of 10 sec? Initially there were also the default value and probably was 10

Reply:
That's what Osmosis was using in the examples, 1 minute might be a lot no? The entire swap + IBC transfer would be significantly slowed down right?

Reply:
This is just an upper bound. Would be slower on the Osmosis side but I did't have tests to know what's the difference in time between a swap with twap 10s and one with 60s. But yeah, we need to make a comparison during e2e tests.